### PR TITLE
tutorials/install: Added Windows instructions

### PR DESCRIPTION
**Pull request by peci1**
# 🦟 Bug fix

Fixes https://github.com/gazebosim/gazebo_test_cases/issues/1186

## Summary
Added Windows install instructions.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by peci1** [comment]
Also, the headings for Ubuntu say Ubuntu 20.04+. However, I don't think that platform is supported for Ionic...

File: tutorials/install.md [hunk 1]
```diff
@@ -50,6 +51,27 @@ installing Gazebo from source.
   brew install gz-sim<#>
   ```
 
+### Windows
+
+First, follow the [gz-cmake](https://gazebosim.org/api/cmake/4/install.html) tutorial for installing Conda, Visual Studio, CMake, and other prerequisites, and also for creating a Conda environment.
```
Comment (azeey) [inline tutorials/install.md:56]: https://github.com/gazebosim/docs/pull/512/ added more detail instructions for installing conda. Should we link to https://gazebosim.org/docs/ionic/install_windows_src/#install-dependencies instead?

**Review by azeey — APPROVED** [review]
One small suggestion, otherwise, LGTM!

File: tutorials/install.md (hunk 1 above)
Comment (peci1) [reply tutorials/install.md]: Great idea! I've updated this PR.

**Pull request merged**